worker/util/Handlebars.js

```diff
@@ -73,4 +79,15 @@ Handlebars.registerHelper('sort', function (array, propertyPath, order) {
   }
 });
 
+
+Handlebars.registerHelper('randNum', function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+});
+
+Handlebars.registerHelper('randList', function (options) {
+  const choices = options.split(',').map(s => s.trim());
```

Comment:
why bother with the split? couldn't you make options an array then have
{{ randList "foo" "bar" "baz,boo,foo" }} then it could have commas?

Reply:
Good shout! 

https://github.com/CatalysmsServerManager/7-days-to-die-server-manager/pull/610

Thanks :smile: